package.json

```diff
@@ -0,0 +1,155 @@
+{
+  "name": "message-handling",
+  "version": "1.0.0",
+  "description": "A rabbitmq implementation for message handling",
```

Comment:
"Lightweight message handling infrastructure."
RabbitMq is just one implementation at the moment.